src/main/java/org/fogbowcloud/saps/engine/core/database/Catalog.java

```diff
@@ -4,97 +4,55 @@
 import java.sql.Timestamp;
 import java.util.Date;
 import java.util.List;
-import java.util.Map;
-
 import org.fogbowcloud.saps.engine.core.dispatcher.notifier.Ward;
 import org.fogbowcloud.saps.engine.core.model.SapsImage;
 import org.fogbowcloud.saps.engine.core.model.SapsUser;
 import org.fogbowcloud.saps.engine.core.model.enums.ImageTaskState;
 
 public interface Catalog {
 
-	public String NONE = "None";
-	public int UNLIMITED = -1;
+	String NONE = "None";
+	int UNLIMITED = -1;
```

Comment:
why all these vars from line 15 to 23 are declared in the interface? aren't they implementation specific?

Reply:
All vars from line 15 to 23 was moved to implementation class